### PR TITLE
Added date functionality.

### PR DESCRIPTION
**Pull request by Highshirt99**

### CODE REVIEW

File: src/reduxStore/todoSlice.js [hunk 1]
```diff
@@ -12,12 +12,21 @@ export const todoSlice = createSlice({
   initialState,
   reducers: {
     addTodo: (state, action) => {
-      state.todos.unshift({ ...action.payload, status: "active" });
+      const date = new Date();
+      const year = date.getFullYear();
+      const month = date.getMonth()+ 1;
+      const day = date.getDate() 
+
+      const dateCreated = `${day}/${month}/${year}`;
```
Comment (codarbind) [inline src/reduxStore/todoSlice.js]: rename `dateCreated` to `created_at`

Comment (codarbind) [reply src/reduxStore/todoSlice.js]: also add `completed_at`

File: src/components/TodoList.js [hunk 2]
```diff
@@ -75,7 +75,10 @@ function TodoList() {
                   <img src={check} alt="" />
                 ) : null}
               </div>
-              {item.title}
+              <p>{item.title}</p>
```
Comment (codarbind) [inline src/components/TodoList.js]: I dont think this is semantic

could there be a better tag/element to use instead of `<p>`?

Comment (codarbind) [reply src/components/TodoList.js]: where would we place/show date of completion?

I think there could be a little not too tiny bar under each todo that could show date of creation and date of completion and perhaps other similar details

let me know which arrangement you want to go with befoe implementation

well done

**Review by codarbind — CHANGES_REQUESTED** [review]

File: src/reduxStore/todoSlice.js (hunk 1 above)
Comment (codarbind) [reply src/reduxStore/todoSlice.js:20]: image

File: src/components/TodoList.js (hunk 2 above)
Comment (codarbind) [reply src/components/TodoList.js]: something like the position where the due date  is here

image

**Comment by Highshirt99** [comment]
All reviews would be made later today. I'm currently at my IT placement.

**Comment by mo-renike** [comment]
I think It is best to add a snapshot of your frontend work to your PR.

**Comment by Highshirt99** [comment]
There's a link to preview each PR made.

**Review by mo-renike — APPROVED** [review]

**Review by codarbind — APPROVED** [review]

**Pull request merged**